modules/RTC/RTC.js

```diff
@@ -223,8 +193,8 @@ export default class RTC extends Listenable {
      * @param {Array<Object>} tracksInfo
      * @returns {Array<JitsiLocalTrack>}
      */
-    static newCreateLocalTracks(tracksInfo) {
-        return _newCreateLocalTracks(tracksInfo);
+    static createLocalTracks(tracksInfo) {
```

Comment:
Need to call RTC#createLocalTracks here because of the above change - https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/proxyconnection/ProxyConnectionService.js#L279

Reply:
👍